docs/user/environment-variables.md

```diff
@@ -38,6 +38,8 @@ The installer accepts a number of environment variable that allow the interactiv
 
 ## Platform-Specific
 
+* `AWS_PROFILE`:
+     The AWS profile that corresponds to value in `${HOME}/.aws/credentials`.  If not provided, the default is "default".
 * `OPENSHIFT_INSTALL_AWS_REGION`:
```

Comment:
If this is not set, aws-sdk-go and/or terraform? defaults to "default".  I currently use "openshift-dev" profile.  exporting `AWS_PROFILE=openshift-dev` results in picking up correct aws credentials.